data/component-design-tokens/badge-design-tokens.json

```
@@ -14,83 +19,167 @@
     "type": "fontSizes",
     "description": "14px"
   },
+  "badge-font-size-large": {
+    "value": "{fontSize.3}",
+    "type": "fontSizes",
+    "description": "16px"
+  },
   "badge-line-height": {
     "value": "{lineHeight.1}",
     "type": "lineHeights",
     "description": "20px"
   },
+  "badge-line-height-large": {
+    "value": "{lineHeight.2}",
+    "type": "lineHeights",
+    "description": "22px"
+  },
   "badge-icon-size": {
-    "value": "{iconSize.1}",
+    "value": "{iconSize.s}",
     "type": "sizing",
-    "description": "16px"
+    "description": "18px"
+  },
+  "badge-icon-size-large": {
+    "value": "{iconSize.m}",
+    "type": "sizing",
+    "description": "20px"
   },
   "badge-border-radius": {
-    "value": "{borderRadius.m}",
+    "value": "{borderRadius.s}",
     "type": "borderRadius",
-    "description": "4px"
+    "description": "6px"
+  },
+  "badge-border-width": {
+    "value": "{borderWidth.s}",
+    "type": "borderWidth",
+    "description": "1px"
+  },
+  "badge-height": {
+    "value": "24px",
+    "type": "sizing"
+  },
+  "badge-height-large": {
+    "value": "32px",
+    "type": "sizing"
   },
   "badge-info-color-bg": {
-    "value": "{color.greyscale.100}",
+    "value": "{color.info.default}",
     "type": "color"
   },
   "badge-info-color-content": {
+    "value": "{color.info.light}",
+    "type": "color"
+  },
+  "badge-info-subtle-color-bg": {
+    "value": "{color.info.light}",
+    "type": "color"
+  },
+  "badge-info-subtle-color-content": {
     "value": "{color.info.default}",
```

Dark colour needed, #00527C
(See comment regarding global tokens)
```suggestion
    "value": "{color.info.textDark}",
```